packages/data/src/atomic-store/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import { mapValues } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { createDerivedAtom } from '@wordpress/stan';
+
+/**
+ *
+ * @param {string}                                     name   Store name.
+ * @param {import('../types').WPDataAtomicStoreConfig} config Atomic store config.
+ * @return {import('../types').WPDataStore} Store.
+ */
+export default function createAtomicStore( name, config ) {
+	return {
+		name,
+		instantiate: ( registry ) => {
+			// I'm probably missing the atom resolver here
```

Comment:
Is it still missing or we can remove this comment? 😄

Reply:
it's fine now  :)